File: 018-photo-storage/photo_storage.py
import math

def number_of_photos(size, drive_capacity):
  return math.floor((drive_capacity * 1000) / size)

print(number_of_photos(1, 1))
print(number_of_photos(2, 1))
print(number_of_photos(4, 256))
print(number_of_photos(3.5, 750))
print(number_of_photos(3.5, 5.5))

File: 018-photo-storage/photoStorage.js
function numberOfPhotos(size, driveCapacity) {
  return Math.floor((driveCapacity * 1000) / size);
}

console.log(numberOfPhotos(1, 1));
console.log(numberOfPhotos(2, 1));
console.log(numberOfPhotos(4, 256));
console.log(numberOfPhotos(3.5, 750));
console.log(numberOfPhotos(3.5, 5.5));